csxtools/image/stack.py

```diff
@@ -112,3 +113,60 @@ def stackstderr(array):
     """
     X, Y = extimage.stackprocess(array, 4)
     return X, Y
+
+
+def ccdmean(images):
+    """Cacluate the mean ccd counts per event
+
+    This function calculates the mean of ccd counts for each event. The
+    input is a "slicerator" object returned by get_fastccd_images.
+
+    Parameters
+    ----------
+    slicerator object : generator returning pims images
+        This is the output of get_fastccd_images
+
+    Returns
+    -------
+    array: 1D numpy array
+    """
+    return np.array([np.nanmean(stackmean(image)) for image in images])
+
+
+def ccdsum(images):
```

Comment:
Also, i like the idea, but we should think of a better name than ``cadsum`` perhaps ``images_sum`` and ``images_mean``?